CMakeLists.txt

```diff
@@ -82,7 +82,9 @@ if(ENABLE_LOGGING)
     include_directories(${Glog_INCLUDE_PATH})
   endif()
   if(Glog_STATIC)
-    add_definitions(-DGOOGLE_GLOG_DLL_DECL=)
+    add_definitions(-DGLOG_NO_EXPORT)
```

Comment:
Have you checked CI results on all platforms?
In my attempt, the same error was still reported if `GLOG_EXPORT` is undefined.

I've tried to follow the bazel configuration at `lotem:master` but fighting cmake config.

Reply:
no, I'm just giving it a try and see the CI results. I'll take a look at the error message now.

Reply:
seems you've succeeded! congrats

Reply:
I need your C++ code changes :-<

Reply:
just take it :D